source/codegen/metadata/nidigital/functions.py

```diff
@@ -2622,5 +2622,26 @@
             }
         ],
         "returns": "ViStatus"
+    },
+    'WriteStatic': {
```

Comment:
@ni-pkhare , I had thought that there were a couple of APIs to be added. But if its just one API, then its okay to add this in the previous PR itself. Sorry for the confusion.

Reply:
You could cherry-pick these 2 commits in that other PR branch.

Reply:
I dont have any strong opinion. You could take a call whether it can be separate PR or same PR.